project/ProjectPlugin.scala

```diff
@@ -1,48 +1,114 @@
-import dependencies.DependenciesPlugin.autoImport.depUpdateDependencyIssues
+//import dependencies.DependenciesPlugin.autoImport.depUpdateDependencyIssues
 import sbt.Keys._
 import sbt.Resolver.sonatypeRepo
-import sbt.ScriptedPlugin._
-import sbt._
-import sbtorgpolicies.OrgPoliciesKeys.orgAfterCISuccessTaskListSetting
-import sbtorgpolicies.OrgPoliciesPlugin
-import sbtorgpolicies.OrgPoliciesPlugin.autoImport._
-import sbtorgpolicies.runnable.syntax._
-import sbtorgpolicies.templates.badges._
+import sbt.ScriptedPlugin.autoImport._
+import sbt.{Def, _}
+// import sbtorgpolicies.OrgPoliciesKeys.orgAfterCISuccessTaskListSetting
+// import sbtorgpolicies.OrgPoliciesPlugin
+// import sbtorgpolicies.OrgPoliciesPlugin.autoImport._
+// import sbtorgpolicies.runnable.syntax._
+// import sbtorgpolicies.templates.badges._
+// import sbtorgpolicies.model.scalac
 
 object ProjectPlugin extends AutoPlugin {
 
-  override def requires: Plugins = OrgPoliciesPlugin
+  // override def requires: Plugins = OrgPoliciesPlugin
 
   override def trigger: PluginTrigger = allRequirements
 
+  object sbtV {
+    val `0.13`: String = "0.13.16"
+    val `1.0`: String  = "1.0.1"
+
+    val crossSbtVersions: List[String] = List(`0.13`, `1.0`)
+  }
+
+  object scalac {
+
+    val `2.10`: String = "2.10.6"
+    val `2.11`: String = "2.11.11"
+    val `2.12`: String = "2.12.3"
+    val `2.13`: String = "2.13.0-M1"
+
+    val latestScalaVersion: String = `2.12`
+
+    val crossScalaVersions: List[String] = List(`2.11`, `2.12`)
+
+  }
```

Comment:
Better name for `sbtV`? I originally had `sbt` but that might inadvertently clash with the `_root_.sbt`

These will be removed once we have a snapshot published and use the corresponding `sbtV` and `scalac` in `model.scala`.

Reply:
Probably, can't think a better one now so I'm good keeping it as it is for now ;)